src/utils/logger.ts

```diff
@@ -0,0 +1,26 @@
+import * as vscode from "vscode";
+import { OutputChannel } from "vscode";
+
+export class Logger {
+  // eslint-disable-next-line no-use-before-define
+  private static instance: Logger;
+
+  private outputChannel: OutputChannel;
+
+  private constructor() {
+    this.outputChannel = vscode.window.createOutputChannel("Tabnine");
+  }
+
+  public static log(message: string): void {
```

Comment:
consider to add a different log level output support (error/info)

Reply:
with a different prefix?

Reply:
ok, cool. will do